packages/widget.client-react/src/context.ts

```diff
@@ -25,6 +26,10 @@ import {
 } from "@osdk/widget.client";
 import React, { useContext } from "react";
 
+export type ExtractObjectSet<OT extends ObjectType> = NonNullable<
```

Comment:
Should we keep this internal for now if it's not reused?

```suggestion
type ExtractObjectSet<OT extends ObjectType> = NonNullable<
```